frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -234,20 +237,72 @@ export const AboutResourcePage = ({
           }
           onChangeValue={handleChangeTranslationValues}
           usesTextArea={translationType === 'description'}
+          showErrors={showAllErrors}
         />
       </div>
     );
   };
 
+  /**
+   * Maps the language key to the text
+   */
+  const getLanguage = (val: 'nb' | 'nn' | 'en') => {
+    if (val === 'nb') return 'Bokmål';
+    if (val === 'nn') return 'Nynorsk';
+    return 'Engelsk';
+  };
+
+  /**
+   * Gets the correct text to display for input fields with missing value
+   *
+   * @param val the value
+   * @param isTitle if the field is title or description
+   */
+  const getMissingInputLanguage = (val: LanguageStringType, isTitle: boolean) => {
+    const valArr: ('nb' | 'nn' | 'en')[] = [];
+    const type = isTitle ? 'tittel' : 'beskrivelse';
+
+    // Add the different languages
+    if (val.nb === '') {
+      valArr.push('nb');
+    }
+    if (val.nn === '') {
+      valArr.push('nn');
+    }
+    if (val.en === '') {
+      valArr.push('en');
+    }
+
+    // Return different messages based on the length
+    if (valArr.length === 1) {
+      return `Du mangler oversettelse for ${type} på ${getLanguage(valArr[0])}.`;
+    }
+    if (valArr.length === 2) {
+      return `Du mangler oversettelse for ${type} på ${getLanguage(valArr[0])} og
+      ${getLanguage(valArr[1])}.`;
+    }
+    if (valArr.length === 3) {
+      return `Du mangler oversettelse for ${type} på ${getLanguage(valArr[0])},
+      ${getLanguage(valArr[1])} og ${getLanguage(valArr[2])}.`;
```

Comment:
Couldn't this message have been constructed in a loop instead of three separate ifs of nearly duplicate code?